CMakeLists.txt

```diff
@@ -2658,7 +2681,7 @@ if(SDL_STATIC)
   add_library(SDL2-static STATIC ${SOURCE_FILES})
   # alias target for in-tree builds
   add_library(SDL2::SDL2-static ALIAS SDL2-static)
-  if (NOT SDL_SHARED OR NOT WIN32)
+  if (NOT SDL_SHARED OR NOT WIN32 OR MINGW)
```

Comment:
is WIN32 not defined when MINGW is?

Reply:
`WIN32` is defined by MinGW toolchains, however the condition should be run either on non-Windows platforms, or on MinGW. I can add brackets if that makes things clearer.

Reply:
OK, I take it back